patroni/ha.py

```diff
@@ -562,11 +568,11 @@ def enforce_master_role(self, message, promote_message):
             # Inform the state handler about its master role.
             # It may be unaware of it if postgres is promoted manually.
             self.state_handler.set_role('master')
-            self.process_sync_replication()
+            self.process_sync_replication(self.patroni.config['synchronous_node_count'])
             self.update_cluster_history()
             return message
         elif self.state_handler.role == 'master':
-            self.process_sync_replication()
+            self.process_sync_replication(self.patroni.config['synchronous_node_count'])
```

Comment:
We could do just `sync_node_count = self.patroni.config['synchronous_node_count']` in the `process_sync_replication()` method.

Reply:
fixed